sql/win_tzname_data.h

```diff
@@ -69,7 +69,7 @@
 {L"Libya Standard Time","Africa/Tripoli"},
 {L"Namibia Standard Time","Africa/Windhoek"},
 {L"Arabic Standard Time","Asia/Baghdad"},
-{L"Turkey Standard Time","Europe/Istanbul"},
+{L"Türkiye Standard Time","Europe/Istanbul"},
```

Comment:
Nope. Windows does not have "Türkiye Standard Time" time zone name. It has "Turkey Standard Time".
![image](https://user-images.githubusercontent.com/1708204/217749731-19fa0bb8-b12d-44fa-a35a-f523b1909fb6.png)

Besides, that file is generated from Unicode consortium data